docs/java-application-development/development-environment-setup.md

```diff
@@ -0,0 +1,167 @@
+# Development Environment Setup
+
+This document describes how to set up the development environment for Kura, which consists of the following components:
+
+* JVM (Java JDK SE 8)
+
+* Eclipse IDE
+
+* Kura Workspace setup
+
+The Kura development environment may be installed on Windows, Linux, or Mac OS. The setup instructions will be the same across each OS though each system may have unique characteristics.
+!!! info
+    The local emulation of Kura code is only supported in Linux and Mac, not in Windows.
+
+## JVM Installation
+
+Download and install JDK SE 8 from the following links as appropriate for your OS.
+
+For Windows and Linux users, the JDK can be downloaded from the following link: [Java SE 8 Downloads](https://adoptium.net/temurin/releases/?version=8).  Use the latest version of Java SE Development Kit and download the version appropriate for your system.
+
+For additional information regarding the installation of Java 8 on all supported operating systems, see [JDK 8 and JRE 8 Installation Guide](https://adoptium.net/installation).
+
+
+## Eclipse IDE
+
+The Eclipse IDE is an open source development tool that consists of an integrated development environment (IDE) and a plug-in system for managing extensions.
+
+### Installing Eclipse
+
+Before installing Eclipse, you should choose directory locations for the Eclipse install and its workspaces.
+
+!!! info
+    The following points should be kept in mind regarding Eclipse installs and workspaces: <br />
+    - The directory location of the Eclipse workspaces should be chosen carefully. 
```

Comment:
A blank line must separate the beginning of the list to have it recognized in markdown. Please remove the `<br />` tags.